CONFIGDOC.md

```diff
@@ -1,18 +1,18 @@
 You can use the Jenkins&trade; plugin for MATLAB&reg; in freestyle and multi-configuration projects. This guide demonstrates how to run MATLAB code on Jenkins, considering a Microsoft&reg; Windows&reg; platform.
 
--  [Use MATLAB for Build](#use-matlab-for-build)
+-  [Add MATLAB to System Path](#add-matlab-to-system-path)
 -  [Specify Build Steps](#specify-build-steps)
       - [Run MATLAB Tests](#run-matlab-tests)
       - [Run MATLAB Command](#run-matlab-command)
 -  [Set Up Freestyle Project](#set-up-freestyle-project)
 -  [Set Up Multi-Configuration Project](#set-up-multi-configuration-project)
 
-## Use MATLAB for Build
-Once you install the plugin on Jenkins, the **Use MATLAB version** checkbox is displayed in the **Build Environment** section of the project configuration window. Selecting the checkbox instructs Jenkins to use MATLAB for your build.
+## Add MATLAB to System Path

```

Comment:
I actually prefer the title you had before. Conceptually they are specifeicaying which matlab they want to use, and its in the details that they learn this means adding it to the PATH

Reply:
Sure. Reverted to the previous title.